.github/workflows/cml-with-dvc.yaml

```diff
@@ -19,25 +18,31 @@ jobs:
         run: |
           cml runner \
               --cloud=aws \
-              --labels=cml-runner \
               --cloud-region=eu-west \
               --cloud-type=t2.micro \
```

Comment:
`t2.micro` isn't particularly great compared to native GHA. Maybe use [something more beefy](https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task#machine-type)?

```suggestion
              --cloud-type=m \
```